ki-ujep/facilities
# feature/organizations

- Add a boolean field for partner organization to the faculty model.
- Create a new section in the admin panel specifically for organizations.
- Develop separate forms for adding and editing organizations in the admin panel.
- Implement a proxy view with the URL "/organization" to handle the display of organization-related content.
- Faculty field label in device admin changed to "Faculty / Organization" 

### How to test
- Test that organizations and faculties are separated in admin
- Check admin fields (Devices, Faculties, Orgs)
- Check URL for all orgs link are `/organization` instead of `/faculty`
